Lateral movement detection package (#4833)
elastic/integrations: packages/lmd/docs/README.md

# Lateral Movement Detection Model

> The Lateral movement detection model package contains the lateral movement detection model and the associated assets. This package requires a Platinum subscription. Please ensure that you have a Trial or Platinum level subscription installed on your cluster before proceeding. This package is licensed under Elastic License v 1.0.

Reviewer: ```suggestion
The Lateral movement detection model package contains assets to detect lateral movement based on file transfer activity. This package requires a Platinum subscription. Please ensure that you have a Trial or Platinum level subscription installed on your cluster before proceeding. This package is licensed under Elastic License v 1.0.
```

> The Lateral movement detection model package contains assets to detect lateral movement based on file transfer activity. This package requires a Platinum subscription. Please ensure that you have a Trial or Platinum level subscription installed on your cluster before proceeding. This package is licensed under Elastic License v 1.0.

Reviewer 2: ```suggestion
The Lateral movement detection model package contains assets that detect lateral movement based on file transfer activity. This package requires a Platinum subscription. Please ensure that you have a Trial, Platinum, or Enterprise subscription before proceeding. This package is licensed under Elastic License v 1.0.
```

## Configuration

To download the assets, click **Settings** > **Install Lateral Movement Detection assets**. 

> Then use these detection rules and anomaly detection jobs for lateral movement detection.

Reviewer 2: ```suggestion
```
I suggest removing this line because it isn't part of the configuration instructions, and if I'm understanding correctly, the following sections describe it in more detail anyway.

### Enable Security rules

> This model uses both anomaly detection and security rules to detect lateral movement in the network. To maximize the benefit of both frameworks, you should consider enabling the list of all "Security Detection Rules" described below. The first four rules are triggered when certain conditions for the anomaly detection jobs are satisfied. The last two rules are behavioral and independent of anomaly detection jobs. See the [documentation](https://www.elastic.co/guide/en/security/current/detection-engine-overview.html) for more information on importing and enabling the rules.

Reviewer 3: ```suggestion
This model uses both anomaly detection and security rules to detect lateral movement in the network. In order to see all alerts detected by this model, you need to enable all the "Security Detection Rules" in the table, as described below. The first four rules are triggered when certain conditions for the anomaly detection jobs are satisfied. The last two rules are behavioral and independent of anomaly detection jobs. See the [documentation](https://www.elastic.co/guide/en/security/current/detection-engine-overview.html) for more information on importing and enabling the rules.
```
